cibuildwheel/linux.py

```diff
@@ -395,8 +386,8 @@ def build(options: Options, tmp_path: Path) -> None:  # noqa: ARG001
             check=True,
             stdout=subprocess.DEVNULL,
         )
-    except subprocess.CalledProcessError:
-        print(
+    except subprocess.CalledProcessError as e:
```

Comment:
nitpick: use a better variable name? Like `proc_err`, maybe?

Reply:
I don't see a huge benefit, there's only one statement in the `except` clause. I will standardise it to look like the others though.